src/backend/backend.h

```diff
@@ -249,37 +281,11 @@ namespace snmalloc
       }
 
       MetaEntry t(meta, remote, sizeclass);
-
-      for (address_t a = address_cast(p);
-           a < address_cast(pointer_offset(p, size));
-           a += MIN_CHUNK_SIZE)
-      {
-        pagemap.set(a, t);
-      }
+      pagemap.set_meta_data(address_cast(p), size, t);
       return {p, meta};
     }
 
-    /**
-     * Get the metadata associated with a chunk.
-     *
-     * Set template parameter to true if it not an error
-     * to access a location that is not backed by a chunk.
-     */
-    template<bool potentially_out_of_range = false>
-    static const MetaEntry& get_meta_data(address_t p)
-    {
-      return pagemap.template get<potentially_out_of_range>(p);
-    }
-
-    /**
-     * Set the metadata associated with a chunk.
-     */
-    static void set_meta_data(address_t p, size_t size, MetaEntry t)
-    {
-      for (address_t a = p; a < p + size; a += MIN_CHUNK_SIZE)
-      {
-        pagemap.set(a, t);
-      }
-    }
+    SNMALLOC_REQUIRE_CONSTINIT
+    static inline Pagemap pagemap;
```

Comment:
Why does this exist?  Things in `Backend` are not supposed to have any per-instance state (other than `LocalState`).  `Pagemap` appears to follow this rule: all of its methods are `static` and all of its state is in a `static` field, so what is this instance for?  Should things not just be calling `Backend::Pagemap::set_metaentry` and friends?